doc/UltiSnips.txt

```diff
@@ -267,27 +268,45 @@ following to your vimrc file or switching to a plugin like Supertab or
 YouCompleteMe. >
    inoremap <c-x><c-k> <c-x><c-k>
 
+If you prefer to use the same trigger for expanding snippets and jumping
+forward, g:UltiSnipsExpandOrJumpTrigger and g:UltiSnipsJumpOrExpandTrigger
+variables are what you want. Try to set g:UltiSnipsExpandOrJumpTrigger if
+you give priority to expanding snippets. It'll expand snippets first when
+you are in a position where you can both expand snippets and jump forward. >
+   let g:UltiSnipsExpandOrJumpTrigger = "<tab>"
+
+Or else, if you have a preference for jumping forward first, set
+g:UltiSnipsJumpOrExpandTrigger instead. >
+   let g:UltiSnipsJumpOrExpandTrigger = "<tab>"
+
+If both g:UltiSnipsExpandOrJumpTrigger and g:UltiSnipsJumpOrExpandTrigger
+is set, g:UltiSnipsExpandOrJumpTrigger variable will take effects.
+
 3.2.2 Using your own trigger functions           *UltiSnips-trigger-functions*
 
 For advanced users there are four functions that you can map directly to a
```

Comment:
no longer four - please just change to "there are functions" which is more future proof

Reply:
Fixed.